src/main/java/com/mercadopago/client/order/OrderClient.java

```diff
@@ -77,4 +80,74 @@ public Order create(OrderCreateRequest request) throws MPException, MPApiExcepti
         return this.create(request, null);
     }
 
+    /**
+     * Method responsible for obtaining order by id
+     *
+     * @param id orderId
+     * @return order response
+     * @throws MPException an error if the request fails
+     * @throws MPApiException an error if the request fails
+     */
+    public Order get(String id) throws MPException, MPApiException {
+        return this.get(id, null);
+    }
+
+    /**
+     * Method responsible for obtaining order by id with request options
+     *
+     * @param id orderId
+     * @param requestOptions metadata to customize the request
+     * @return order response
+     * @throws MPException an error if the request fails
+     * @throws MPApiException an error if the request fails
+     */
+    public Order get(String id, MPRequestOptions requestOptions) throws MPException, MPApiException {
+        LOGGER.info("Sending order get request");
+
+        MPResponse response =
+                send(String.format(URL_WITH_ID, id), HttpMethod.GET, null, null, requestOptions);
+
+        Order result = Serializer.deserializeFromJson(Order.class, response.getContent());
+        result.setResponse(response);
+
+        return result;
+    }
```

Comment:
Adicionar validação assim como fez no process e modificar usando StringUtils, pois valida se é null e vazio, inclusive se há somente espaço na string.
```suggestion
    public Order get(String id, MPRequestOptions requestOptions) throws MPException, MPApiException {
        LOGGER.info("Sending order get request");
        
        if (StringUtils.isBlank(id)) {
            throw new IllegalArgumentException("Order id cannot be null or empty");
        }

        String url = String.format(URL_WITH_ID, id);
        MPResponse response = send(url, HttpMethod.GET, null, null, requestOptions);

        Order order = Serializer.deserializeFromJson(Order.class, response.getContent());
        order.setResponse(response);

        return order;
    }
```